### PR TITLE
Add logging with null logger

### PR DESCRIPTION
**Pull request by brandondunne**
Depends on https://github.com/ManageIQ/linux_admin/pull/112

### CODE REVIEW

File: lib/linux_admin/common.rb [hunk 1]
```diff
@@ -11,6 +17,7 @@ def run(cmd, options = {})
     end
 
     def run!(cmd, options = {})
+      AwesomeSpawn.logger = logger
```
Comment (Fryguy) [inline lib/linux_admin/common.rb]: This is weird to do it on every call to run.  Not sure where you would do it otherwise though.

**Comment by brandondunne** [comment]
@Fryguy Please review :icecream:

File: lib/linux_admin/disk.rb [hunk 2]
```diff
@@ -5,8 +5,10 @@
 
 require 'linux_admin/partition'
 
-class LinuxAdmin
-  class Disk < LinuxAdmin
+module LinuxAdmin
+  class Disk
+    include Common
```
Comment (Fryguy) [inline lib/linux_admin/disk.rb]: :tada:

File: lib/linux_admin/package.rb [hunk 3]
```diff
@@ -1,4 +1,7 @@
-class LinuxAdmin
-  class Package < LinuxAdmin
+module LinuxAdmin
+  class Package
+    class << self
+      include Common
+    end
```
Comment (Fryguy) [inline lib/linux_admin/package.rb]: This is just `extend Common`

**Pull request closed without merging**